strongbox-db-schema/src/main/java/org/carlspring/strongbox/db/schema/Vertices.java

```diff
@@ -4,19 +4,30 @@
 {
 
     String ARTIFACT = "Artifact";
+
```

Comment:
@anki2189 I would leave it without emty lines.

Reply:
Fixed.